src/stats.cpp

```diff
@@ -0,0 +1,90 @@
+
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+#include <cmath>
+
+namespace py = pybind11;
+
+double erfinv(double x);
+double normal(double x);
+double truncnorm(double x, double alpha, double beta);
```

Comment:
I suggest calling it `_ppf` (or maybe rather `_inverse_cdf`) to clearify what the functions does

Reply:
Went for _ppf.